source/developer/bot-accounts.rst

```diff
@@ -33,36 +33,37 @@ If you would like to see improvements to bot accounts, `let us know in the Featu
   :local:
 
 Configuration Settings
-------------------------
+----------------------
 
 By default, plugins can create and manage bot accounts. To enable bot account creation through the user interface or the RESTful API:
 
 1. Go to **System Console > Integrations > Bot Accounts**.
 2. Set **Enable Bot Account Creation** to ``true``.
 
-Once set, System Administrators can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
+Once set, System Admin can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
 
 Bot Account Creation
------------------------
+--------------------
 
 Below are different ways to create bot accounts. After the bot account is created, make sure to:
 
 1. Copy the generated bot access token for your integration.
 2. Add the bot account to teams and channels you want it to interact in.
 
 User Interface (UI)
-^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^
 
 1. Go to **Main Menu > Integrations > Bot Accounts**.
-2. Click **Add Bot Account**.
+2. Select **Add Bot Account**.
 3. Set the **Username** of the bot. The username must begin with a letter, and contain between 3 and 22 lowercase characters made up of numbers, letters, and the symbols ".", "-", and "_".
```

Comment:
```suggestion
3. Set the **Username** of the bot. The username must begin with a letter, and contain between 3 and 22 lowercase characters made up of numbers, letters, and symbols including ".", "-", or "_".
```

Reply:
I'm making an assumption here that bot usernames aren't required to include all supported symbols. If they are required to have all 3 supported symbols, then this suggestion is invalid.

Reply:
I agree - based on the bot names I've seen your suggestion is correct.